ui/src/styleVariables.js

```diff
@@ -21,4 +21,6 @@ module.exports = {
   'border-style-base': 'solid',
   'border-color-split': 'rgb(232, 232, 232)',
   'card-radius': '2px',
+  'font-size-base': '14px',
```

Comment:
this maybe actually `1rem` so you don't have to add this, where you need this just use `1rem`